packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -1137,21 +1099,21 @@ class _SearchFieldSuffix extends StatelessWidget {
 }
 
 class SearchNavigationControls extends StatelessWidget {
-  const SearchNavigationControls(this.controller, {@required this.onClose});
+  const SearchNavigationControls(this.controller, {required this.onClose});
 
   final SearchControllerMixin controller;
 
-  final VoidCallback onClose;
+  final VoidCallback? onClose;
 
   @override
   Widget build(BuildContext context) {
     return ValueListenableBuilder(
         valueListenable: controller.searchMatches,
-        builder: (context, matches, _) {
+        builder: (context, List<DataSearchStateMixin> matches, _) {
```

Comment:
another option instead of adding the type here, is to add the type to `ValueListenableBuilder<List<DataSearchStateMixin>>`